worksheet_A/TerminalHacking.cpp

```diff
@@ -4,8 +4,31 @@
 #include "stdafx.h"
 #include "WordList.h"
 
-const int wordLength = 5;
+
+const int wordlength = 5;
 const int numberOfWords = 15;
+int lives = 4;
+bool correctguess;
+int LikenessScore = 0;
+std::string input;
```

Comment:
Do these need to be global variables? Would probably be better to declare them inside `main()`.